pkg/logql/metrics.go

```diff
@@ -335,13 +335,13 @@ func tagsToKeyValues(queryTags string) []interface{} {
 		if len(val) != 2 {
 			continue
 		}
-		vals = append(vals, val...)
+		vals = append(vals, strings.ToLower(val[0]), val[1])
```

Comment:
double check: is it fine to treat "level=warn" different than "level=WARN"? i

Reply:
the only way this is applied is for something we pass in via the X-Query-Tags header, so it's a little hard to reason about what we'll send. previously this was forcing everything to lowercase but I didn't really want that behavior for the value.  in all of our other logfmt the key is lowercase so it made sense to keep lowercasing the key. but we leave the value along now.